Mods/HMC Vile's Pre-Industrial/Defs/Vile_Mending.xml

```diff
@@ -0,0 +1,294 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+	<ThingDef ParentName="BuildingFueled">
```

Comment:
Nope, this parent has fuel tab that doesn work with exotic fuels, change parentName to parent of BuildingFueled